### PR TITLE
Bump Ruby dependencies up, upgrade CocoaPods to 1.11.3

### PR DESCRIPTION
**Pull request by kreeger**
I noticed we had a handful of outdated Ruby dependencies which were getting flagged on our [Security tab in this repo](https://github.com/oreillymedia/flapjack/security/dependabot), so I've taken the time to update them.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '~> 1.10.0'
+gem 'cocoapods', '~> 1.11.3'
```
Comment (awaltzforvenus) [inline Gemfile:3]: We can probably just kill this now, honestly.

**Review by awaltzforvenus — CHANGES_REQUESTED** [review]
Sorry for missing these before, and thanks for catching them! Just the one comment, otherwise LGTM.

File: Gemfile (hunk 1 above)
Comment (kreeger) [reply Gemfile:3]: We still want to keep this since this is a public library and some folks are still on CocoaPods!

Comment (awaltzforvenus) [reply Gemfile:3]: I....thought this was a PR on a different project when I made that comment. 😂🤦🏻‍♂️ Yep, you're right.

**Review by awaltzforvenus — APPROVED** [review]
I'm good with this!

**Pull request merged**